analyses/fusion_filtering/README.md

```diff
@@ -0,0 +1,43 @@
+We also annotated fusions between pairs of genes that were observed in TCGA.
+We also gather counts for recurrent fusions and fused genes found in more than 3 participants per histology and represent them as binary matrices per sample.
+
+#### Inputs from data download
+* pbta-fusion-starfusion.tsv.gz : aggregated starfusion calls ; a column tumor_id with the samples BS ID is added to each sample files
+* pbta-fusion-arriba.tsv.gz : aggregated arriba calls; a column tumor_id with the samples BS ID is added to each sample files ; a column annots is added from running FusionAnnotator
+* pbta-gene-expression-rsem-fpkm.polya.rds : aggregated polya samples fpkm data
+* pbta-gene-expression-rsem-fpkm.strande.rds : aggregated stranded fpm data
+
+#### Inputs used as reference
+* genelistreference.txt : known kinases, oncogenes, tumor suppressors, curated transcription factors [@doi:10.1016/j.cell.2018.01.029], COSMIC census genes. MYBL1 [@doi:10.1073/pnas.1300252110], SNCAIP [@doi:10.1038/nature11327], FOXR2 [@doi:10.1016/j.cell.2016.01.015], TTYH1 [@doi:10.1038/ng.2849], and TERT [@doi:10.1038/ng.3438; @doi:10.1002/gcc.22110; @doi:10.1016/j.canlet.2014.11.057; @doi:10.1007/s11910-017-0722-5] were added to the oncogene list and BCOR [@doi:10.1016/j.cell.2016.01.015] and QKI [@doi:10.1038/ng.3500] were added to the tumor suppressor gene list based on pediatric cancer literature review
+* fusionreference.txt :  known TCGA fusions
+* Brain_FPKM_hg38_matrix.txt.zip : GTex brain samples FPKM data
+
+#### Outputs saved to data download
+* pbta-fusion-putative-oncogenic.tsv
+* pbta-fusion-recurrently-fused-genes-byhistology.tsv
+* pbta-fusion-recurrently-fused-genes-bysample.tsv
+
+### Run script
+`bash run_fusion_merged.sh` 
+
+#### Order of scripts in analysis
+`01-fusion-standardization.R` : Standardizes fusion calls from STARFusion and Arriba
+
+`02-fusion-filtering.R` : Artifact filtering by removing readthroughs and NEIGHBORS from annots column; annots column also contains red flag databases that are used to further filter common/normal occuring fusions; all fusions where both genes are expressed <1TPM is removed as non-expressed fusions; fusions required to have atleast 1 JunctionSpanningRead minus  SpanningFragCount-JunctionReadCount to be <10
+
+`03-Calc-zscore-annotate.R` : Calculates zscore for gene fused gene's expression compared to GTeX brian samples and annotates if differently expressed or not
+
+`04-project-specific-filtering.Rmd` : Notebook to perform project specific filtering. We removed fusions with genes fused more than 5 times in a samples as potential artifact. We kepth fusions that were called by both callers and if >2 samples per histology called the fusion. We then prioritize the fusions as putative-oncogenic fusions if any fused gene in the fusion is annotated as kinases, oncogenes, tumor suppressors, curated transcription factors, cosmic conses gene list or is a TCGA fusions
```

Comment:
Can you change the wording here to match the changes that you've made above? For example, this says `cosmic conses gene list`. It's not clear to me if it should be COSMIC consensus (line 11) or COSMIC census (line 22).

Reply:
right I'm updating that, it should be COSMIC Cancer Gene Census in all places.